### ISSUE TITLE
Add repository link for extra

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#15337]
### Summary
In my opinion, the package information section needs a link to the package repository.

For example, we have found a bug, but how do you report this to the package author?
- We look at the information about the package by clicking the "View details" button in the list of installed packages, the description may contain a link to the repository in the "Readme", or may not.
- Or looking for a package via google or modx.com. The package description on modx.com may contain a link to the repository, or may not :)
- Or we search on github, and it is not a fact that we will find it quickly.

In general, a whole quest to find the original repository. I think a separate link would be helpful.

Or we need to somehow indicate to the authors that the link should be always indicated in the "Readme" of the package.

### Why is it needed?
For quick interaction between users and package authors.

### Suggested solution(s)
In my opinion, the package information section needs a link to the package repository.

![package_repo](https://user-images.githubusercontent.com/12523676/99835408-9bccf280-2b75-11eb-994e-6b9554b6ddf5.png)

Or we need to somehow indicate to the authors that the link should be always indicated in the "Readme" of the package.

**Comment by alroniks** [comment]
Could we move fields a bit? I mean, if we are touching this view, maybe make sense to redesign it a bit, to avoid usage form elements for read-only values, for example, and place information by its priority. For example, technical fields about uploaded, installed dates, etc should be visible, but it is not important information to be shown in the middle of the window.

**Comment by Ruslan-Aleev** [comment]
Yes, I agree with that, I slightly changed the view for the 3.x branch (https://github.com/modxcms/revolution/pull/15193), but we can make it more convenient.

**Comment by JoshuaLuckers** [comment]
@jaygilmore what do you think?

**Comment by JoshuaLuckers** [comment]
@rthrash what do you think about this? It would also mean the field needs to be provided by the package provider.

**Comment by opengeek** [comment]
The link to report problems with a package is up to the package author to provide, and that is in the package provider already. In fact, there are three fields — website_url (required), support_url (optional), and documentation_url (optional) — for this purpose.
